.github/workflows/release.yml

```diff
@@ -35,6 +35,6 @@ jobs:
         run: |
           git config --local user.email imodeljs-admin@users.noreply.github.com
           git config --local user.name imodeljs-admin
-          pnpm publish-packages -y --branch ${{ github.ref_name }} --message "Version bump [skip actions]"
+          pnpm publish-packages -y --branch ${{ github.ref_name }} --message "Version bump [skip actions][Publish Docs]"
```

Comment:
just for consistency I'd prefer `[publish docs]` instead of `[Publish Docs]

Reply:
Fixed